unit_test/blas/Test_Blas_gesv.hpp

```diff
@@ -242,11 +278,13 @@ TEST_F( TestCategory, gesv_float ) {
   }
   #ifdef KOKKOSKERNELS_ENABLE_TPL_BLAS
```

Comment:
Thanks for addressing the feedback. Now that the runtime error is checked for, please clean this code up by removing the ifdefs and if/else branches.

Reply:
Actually, I always tests for both pivoting and no pivoting cases due to the newly added runtime error check. However, I still need these ifdefs and if/else branches for when ```KOKKOSKERNELS_ENABLE_TPL_MAGMA``` is ```ON``` and the view is not in CudaSpace, the tests should not run unless ```KOKKOSKERNELS_ENABLE_TPL_BLAS``` is ```ON```.

Reply:
@e10harvey I removed the ifdefs and if/else branches in the unit test after fixing the "undefined reference" error when not having MAGMA and BLAS. The tests are now clean.